testCommon/src/testFixtures/java/com/virtuslab/gitmachete/testcommon/TestProcessUtils.java

```diff
@@ -22,15 +22,13 @@ public static String runProcessAndReturnStdout(Path workingDirectory, int timeou
     boolean completed = process.waitFor(timeoutSeconds, TimeUnit.SECONDS);
 
     String stdout = IOUtils.toString(process.getInputStream(), StandardCharsets.UTF_8);
-
+    String stderr = IOUtils.toString(process.getErrorStream(), StandardCharsets.UTF_8);
     String commandRepr = Arrays.toString(command);
 
-    if (!completed || process.exitValue() != 0) {
-      System.out.println("Stdout of ${commandRepr}: ${System.lineSeparator()}");
-      System.out.println(stdout);
-      System.err.println("Stderr of ${commandRepr}: ${System.lineSeparator()");
-      System.err.println(IOUtils.toString(process.getErrorStream(), StandardCharsets.UTF_8));
-    }
+    Assert.assertTrue("Stdout of " + commandRepr + " : " + System.lineSeparator() + " " + stdout,
+        !completed || process.exitValue() != 0);
```

Comment:
Whoops the condition should be inverted 😅 
```suggestion
        completed && process.exitValue() == 0);
```